lambda/src/main/scala/pricemigrationengine/services/CohortTableLive.scala

```diff
@@ -175,9 +175,27 @@ object CohortTableLive {
                     _ => Logging.info(s"Wrote $result to Cohort table")
                   )
               }.provide(dependencies)
+
+              override def fetchAll(): IO[CohortFetchFailure, ZStream[Any, CohortFetchFailure, CohortItem]] = {
+                for {
+                  cohortTableConfig <- CohortTableConfiguration.cohortTableConfig
+                    .mapError(error => CohortFetchFailure(s"Failed to get configuration:${error.reason}"))
+                  stageConfig <- StageConfiguration.stageConfig
```

Comment:
`StageConfig` is already available from the impl construction at [line 117](https://github.com/guardian/price-migration-engine/blob/98cac5036bbf612c574b6bdae00a85799b99880f/lambda/src/main/scala/pricemigrationengine/services/CohortTableLive.scala#L117)